analyses/copy_number_consensus_call/src/scripts/merged_to_individual_files.py

```diff
@@ -0,0 +1,114 @@
+## Nhat Duong
+## November, 22 2019
+
+import numpy as np
+import pandas as pd
+import subprocess
+import sys
+import os
+
+######### ASSUMPTIONS ########
+# 1) Files are feed into stdout as manta, cnvkit, then freec
```

Comment:
It looks like this is using the command line arguments so not stdin.

Reply:
I apologize, I will fix that.